enterprise/server/workflow/service/service.go

```diff
@@ -419,6 +420,14 @@ func (ws *workflowService) createActionForWorkflow(ctx context.Context, wf *tabl
 	return actionDigest, err
 }
 
+func (ws *workflowService) workflowsPoolName() string {
+	cfg := ws.env.GetConfigurator().GetRemoteExecutionConfig()
+	if cfg != nil && cfg.WorkflowsPoolName != "" {
+		return cfg.WorkflowsPoolName
+	}
+	return "default"
```

Comment:
refers to https://github.com/buildbuddy-io/buildbuddy/blob/62f1f2cf940a044cc12c3fa478d31129a2bbdcfd/enterprise/server/remote_execution/execution_server/execution_server.go#L440

Reply:
Could we make that "default" a const in https://github.com/buildbuddy-io/buildbuddy/blob/enterprise/server/remote_execution/platform/platform.go?

Reply:
Done, good call